eatin-client/src/homePage/CategoryCarousel.tsx

```diff
@@ -0,0 +1,177 @@
+import { FC } from "react";
+import { Button, Rating } from "@mui/material";
+import CancelIcon from "@mui/icons-material/Cancel";
+import { Carousel, CarouselItem, CarouselProps } from "../components/ui/Carousel";
+import { useNavigate } from "react-router-dom";
+
+export class Recipe {
```

Comment:
I don't think that you should declare all of the params as `string`, some of them are `int` and so on